framework/src/modules/chain/migrations/sql/20191008170600_create_mem_accounts_public_key_index.sql

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+ /*
+  DESCRIPTION: Create index for mem_accounts.publicKey
+  PARAMETERS: None
+*/
+
+CREATE INDEX IF NOT EXISTS "mem_accounts_public_key" ON "mem_accounts" (ENCODE("publicKey", 'hex'));
```

Comment:
I think it will be nice to keep the publicKey out of index since we will use the state store in #4136

Reply:
@shuse2 I agree that maybe creating index for publicKey just for this issue is not necessary but I'd prefer to tackle it after or during working on #4136. Since everything goes to bft branch we can remove this migration file later. But it's up to @lsilvs to decide. This is just my opinion.